### PR TITLE
README: add circleci badge

### PR DESCRIPTION
**Pull request by rootulp**

### CODE REVIEW

**Comment by palantirtech** [comment]
Thanks for your interest in [palantir/godel](https://github.com/palantir/godel), @rootulp! Before we can accept your pull request, you need to sign our contributor license agreement - just visit https://cla.palantir.com/ and follow the instructions. Once you sign, I'll automatically update this pull request.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+[![Circle CI](https://circleci.com/gh/palantir/godel.svg?style=svg)](https://circleci.com/gh/palantir/godel)
```
Comment (nmiyake) [inline README.md]: I prefer the visual style of the badge being below the header like so:

```
# Skylab

[![Circle CI](https://circleci.com/gh/palantir/godel.svg?style=svg)](https://circleci.com/gh/palantir/godel)

 gödel is...
```

File: README.md [hunk 2]
```diff
@@ -1,7 +1,5 @@
-[![Circle CI](https://circleci.com/gh/palantir/godel.svg?style=svg)](https://circleci.com/gh/palantir/godel)
+# gödel [![Circle CI](https://circleci.com/gh/palantir/godel.svg?style=svg)](https://circleci.com/gh/palantir/godel)
```
Comment (nmiyake) [inline README.md]: Prefer style requested in previous feedback:

```
gödel		
=====

[![Circle CI](https://circleci.com/gh/palantir/godel.svg?style=svg)](https://circleci.com/gh/palantir/godel)

gödel is a Go...
```

Obviously just a matter of personal preference (plenty of high-profile public projects use different styles), but I visually prefer it to having the badges precede the header or be part of it. This is also the same as the style used by Kubernetes and feign for their README badges.

**Comment by nmiyake** [comment]
Closed in favor of #29 . Thanks for flagging this and opening the PR!

**Pull request closed without merging**